content/github/writing-on-github/index.md

```diff
@@ -5,6 +5,7 @@ redirect_from:
   - /articles/github-flavored-markdown/
   - /articles/writing-on-github/
   - /categories/writing-on-github
+intro: 'You can structure the information shared on GitHub with various formatting options.'
```

Comment:
```suggestion
intro: 'You can structure the information shared on {% data variables.product.product_name %} with various formatting options.'
```
This looks great @codewithdev 🎉 I made one small change, replacing `GitHub` with `{% data variables.product.product_name %}`. Instead of writing out product names, we use product variables just in case it ever changes. I don't think `GitHub` will ever change, but we use one anyway 😆 

You can see all of the product variables in https://github.com/github/docs/blob/main/data/variables/product.yml